playground/centernet.res18.coco.512size/config.py

```diff
@@ -52,7 +52,7 @@
     SOLVER=dict(
         OPTIMIZER=dict(
             NAME="SGD",
-            BASE_LR=0.02,
+            BASE_LR=0.002,
```

Comment:
please do not change this, thanks.

Reply:
0.02 is too bigger for one GPU, i will change back it.